lib/commands.py

```diff
@@ -149,19 +150,19 @@ def ssh(hostname_or_ip, cmd, check=True, simple_output=True, suppress_fingerprin
     return result_or_exc
 
 def scp(hostname_or_ip, src, dest, check=True, suppress_fingerprint_warnings=True, local_dest=False):
-    options = '-o "BatchMode yes"'
+    opts = '-o "BatchMode yes"'
```

Comment:
Not directly related to the commit message, no? :)

Reply:
Indeed. Commit message now updated.